index.rb

```diff
@@ -144,6 +146,38 @@ def show(**kwargs)
   end
 end
 
+venues << Venue.new(:name => 'Kilowatt', :link => 'https://kilowattbar.com/') do
+  URI.open(URI.join(link, 'events-1')) do |html|
+    # We need to pull the api key from a script tag inside an iframe. Nokogiri
+    # doesn't parse javascript, so a greasy regex will have to do for now.
+    Nokogiri::HTML(html).css('iframe').attr('srcdoc').text.match(/"apiKey":"(?<apikey>\w+)"/) do |match|
+      kw_api_key = match[:apikey]
+
+      if kw_api_key
+        uri = URI.parse('https://events-api.dice.fm/v1/events?page[size]=24&types=linkout,event&filter[venues][]=Kilowatt')
+        response = Net::HTTP.start(uri.host, uri.port, use_ssl: true) do |http|
+          request = Net::HTTP::Get.new(uri)
+          request['Accept'] = 'application/json'
+          request['x-api-key'] = kw_api_key
+
+          http.request(request)
+        end
+
+        if Net::HTTPOK === response
```

Comment:
Same here—I think this should fail somehow if we get a bad response.

Reply:
I removed the if block in favor of getting a messy error from github actions if making the request or parsing the body fails.

If (aka when) we run into issues here, we can beef up error handling